ext/pg_result.c

```diff
@@ -583,24 +606,23 @@ pgresult_nfields(VALUE self)
 
 /*
  * call-seq:
- *    res.fname( index ) -> String
+ *    res.fname( index ) -> String or Symbol
  *
  * Returns the name of the column corresponding to _index_.
```

Comment:
It seems worth mentioning `field_name_type` in these cases where "string or symbol" may be returned.

Reply:
Yes, I'll change this.